### ISSUE TITLE
Warning when activating Push-to-Deploy

### ISSUE THREAD

**Issue by zu** [wppusher/wppusher-documentation#8]
Warning: Invalid argument supplied for foreach() in [path/to/webroot]/wordpress/wp-content/plugins/wppusher/Pusher/Git/BitbucketApiClient.php on line 25

It's only a warning, so I don't know if it's relevant. But thought I'd like to document it.
Maybe it helps.

**Comment by petersuhm** [comment]
Thanks for reporting this!

This is poorly documented in the plugin, but I think this might be because you haven't set up a Bitbucket token, which is needed to set up Push-to-Deploy (even for public plugins). Are you on v2.1.1 of WP Pusher?

**Comment by zu** [comment]
Yes I'm on version 2.1.1 and I believed it when it said "You only need a token if your repositories are private." ;-)
